WORKSPACE

```diff
@@ -140,6 +144,7 @@ load("@rules_python//python:repositories.bzl", "python_register_toolchains")
 
 python_register_toolchains(
     name = "python3_10",
+    ignore_root_user_error = True,
```

Comment:
This was required to avoid the following error:

```
root@px-dev-docker-nas-metal-ubuntu22-l1-kvm:/px/src/px.dev/pixie# bazel build //...
DEBUG: /root/.cache/bazel/_bazel_root/54060b0ed2e63c063d495ae4fb1a7d19/external/rules_python/python/pip.bzl:49:10: pip_install is deprecated. Please switch to pip_parse. pip_install will be removed in a future release.
INFO: Repository python3_10_x86_64-unknown-linux-gnu instantiated at:
  /px/src/px.dev/pixie/WORKSPACE:145:27: in <toplevel>
  /root/.cache/bazel/_bazel_root/54060b0ed2e63c063d495ae4fb1a7d19/external/rules_python/python/repositories.bzl:580:26: in python_register_toolchains
Repository rule python_repository defined at:
  /root/.cache/bazel/_bazel_root/54060b0ed2e63c063d495ae4fb1a7d19/external/rules_python/python/repositories.bzl:404:36: in <toplevel>
INFO: repository @python3_10_x86_64-unknown-linux-gnu' used the following cache hits instead of downloading the corresponding file.
 * Hash '5d0429c67c992da19ba3eb58b3acd0b35ec5e915b8cae9a4aa8ca565c423847a' for https://github.com/indygreg/python-build-standalone/releases/download/20231002/cpython-3.10.13+20231002-x86_64-unknown-linux-gnu-install_only.tar.gz
If the definition of 'repository @python3_10_x86_64-unknown-linux-gnu' was updated, verify that the hashes were also updated.
ERROR: An error occurred during the fetch of repository 'python3_10_x86_64-unknown-linux-gnu':
   Traceback (most recent call last):
        File "/root/.cache/bazel/_bazel_root/54060b0ed2e63c063d495ae4fb1a7d19/external/rules_python/python/repositories.bzl", line 218, column 25, in _python_repository_impl
                fail("The current user is root, please run as non-root when using the hermetic Python interpreter. See https://github.com/bazelbuild/rules_python/pull/713.")
Error in fail: The current user is root, please run as non-root when using the hermetic Python interpreter. See https://github.com/bazelbuild/rules_python/pull/713.
ERROR: /px/src/px.dev/pixie/WORKSPACE:145:27: fetching python_repository rule //external:python3_10_x86_64-unknown-linux-gnu: Traceback (most recent call last):
        File "/root/.cache/bazel/_bazel_root/54060b0ed2e63c063d495ae4fb1a7d19/external/rules_python/python/repositories.bzl", line 218, column 25, in _python_repository_impl
                fail("The current user is root, please run as non-root when using the hermetic Python interpreter. See https://github.com/bazelbuild/rules_python/pull/713.")
Error in fail: The current user is root, please run as non-root when using the hermetic Python interpreter. See https://github.com/bazelbuild/rules_python/pull/713.
ERROR: Error computing the main repository mapping: Encountered error while reading extension file 'requirements.bzl': no such package '@vizier_api_python_deps//': no such package '@python3_10_x86_64-unknown-linux-gnu//': The current user is root, please run as non-root when using the hermetic Python interpreter. See https://github.com/bazelbuild/rules_python/pull/713.
Loading:
```

In order to avoid this, I believe we have two options:
* Use user namespaces to remap the container root user to a non root uid. This would mean our CI and adhoc use of the dev container would need to supply that at runtime.
* Update our packer built images to include an additional user and ensure that all the adhoc and CI use cases work under this new user.

Reply:
Maybe leave a note in the code saying that this is to allow the root user to build the code base which often happens when building in a containerized environment.

Reply:
Updated. Let me know if that doesn't address your feedback.